test/e2e/gardener/common.go

```diff
@@ -86,7 +86,8 @@ func DefaultShoot(name string) *gardencorev1beta1.Shoot {
 				KubeAPIServer: &gardencorev1beta1.KubeAPIServerConfig{},
 			},
 			Networking: &gardencorev1beta1.Networking{
-				Type: pointer.String("calico"),
+				Type:       pointer.String("calico"),
+				IPFamilies: []gardencorev1beta1.IPFamily{gardencorev1beta1.IPFamilyIPv4},
```

Comment:
Why is this needed? Shouldn't that be the default?
Is it only needed for the update test?

Reply:
It actually fixxed the `pull-gardener-e2e-kind-migration` e2e test. For the upgrade test we have a fix coming up hopefully™️. But yes IPv4 is the default, I do like explicit more then implicit though.

If you know why this fixxed the previous mentioned e2e test, please let us know 😆

Reply:
We will drop the commit for now...